### PR TITLE
[CUBRIDMAN-7] manual for CBRD-23649 (dual table) (Korean)

### PR DESCRIPTION
**Pull request by joohok**
http://jira.cubrid.org/browse/CUBRIDMAN-7

Korean manual for dual class
Description of dual class is inserted into system catalog class section.

- definition
- usage
- example

### CODE REVIEW

**Review by hornetmj — APPROVED** [review]

File: ko/sql/catalog.rst [hunk 1]
```diff
@@ -885,6 +885,48 @@ db_ha_apply_info
 | start_time           | DATETIME      | applylogdb 프로세스가 슬레이브 DB에 접속한 시간                             |
 +----------------------+---------------+-----------------------------------------------------------------------------+
 
+dual
+----
+
+dual 테이블은 오직 하나의 열과 행을 가지며 더미 테이블로 사용된다. dual 테이블은 상수, 계산식 또는 SYS_DATE나 USER와 같은 의사 칼럼을 조회할 때 사용된다. 큐브리드에서 의사 칼럼은 함수로써 제공되며 이에 대한 자세한 내용과 예제들은 :ref:`operators-and-functions` 를 참고한다. 상수, 계산식 또는 의사 칼럼을 조회할 때는 FROM절을 생략하여도 dual 테이블이 자동적으로 참조된다. dual 테이블은 다른 시스템 카탈로그처럼 dba 소유로 생성되지만 dba는 dual 테이블에 대해 SELECT 연산만 수행가능하다. 하지만 다른 시스템 카탈로그와 다르게 PUBLIC 사용자들 또한 dual 테이블에 대해 SELECT 연산을 수행할 수 있다.
```
Comment (wrlawodms) [inline ko/sql/catalog.rst:891]: "하지만 다른 시스템 카탈로그와 다르게 PUBLIC 사용자들 또한 dual 테이블에 대해 SELECT 연산을 수행할 수 있다."
Can PUBLIC users do other operations to system catalogs like UPDATE?

Comment (joohok) [reply ko/sql/catalog.rst:891]: No, PUBLIC users can not do any operations on system catalog classes.

Comment (wrlawodms) [reply ko/sql/catalog.rst:891]: This sentence may confuse. What do you think?

Comment (joohok) [reply ko/sql/catalog.rst:891]: "dual 테이블은 시스템 카탈로그이지만, PUBLIC 사용자는 dual 테이블에 대해 SELECT 연산을 수행할 수 있다. "
how about this?

Comment (wrlawodms) [reply ko/sql/catalog.rst:891]: I was confused, and the sentence before this is better to be removed, which makes me confused. However, I know you should consider consistency with the English manual. I leave it to you to decide whether to remove it or not.
```suggestion
dual 테이블은 오직 하나의 열과 행을 가지며 더미 테이블로 사용된다. dual 테이블은 상수, 계산식 또는 SYS_DATE나 USER와 같은 의사 칼럼을 조회할 때 사용된다. 큐브리드에서 의사 칼럼은 함수로써 제공되며 이에 대한 자세한 내용과 예제들은 :ref:`operators-and-functions` 를 참고한다. 상수, 계산식 또는 의사 칼럼을 조회할 때는 FROM절을 생략하여도 dual 테이블이 자동적으로 참조된다. 다른 시스템 카탈로그와 다르게 PUBLIC 사용자들 또한 dual 테이블에 대해 SELECT 연산을 수행할 수 있다.
```

**Review by wrlawodms — APPROVED** [review]

File: ko/sql/catalog.rst (hunk 1 above)
Comment (joohok) [reply ko/sql/catalog.rst:891]: Thanks, I think another PR is needed to remove that sentence in English and Korean manual for consistency.

**Pull request merged**